### ISSUE TITLE
Delete tag from both the repository and remote at the same time

### ISSUE THREAD

**Issue by richardtop** [git-up/GitUp#592]
![image](https://user-images.githubusercontent.com/8013017/65838958-4d291d00-e311-11e9-9f82-b616da1ede7c.png)

Currently when deleting a tag, I have to first "delete from all remotes" and then "delete from the local repository".

I'd like to have a third option "delete from the repository and remotes"

**Comment by lucasderraugh** [comment]
Could work the same as deleting a branch where we prompt to remove from remotes as well.

**Comment by richardtop** [comment]
@lucasderraugh Yes, but I'd appreciate automatic deletion of both.

Currently, it's quite annoying to press "Also delete from the Remote" all the time. I think, there should be an option with a confirmation dialog (to not trigger it accidentally) to delete both the tag and it's remote counterpart.

**Comment by lucasderraugh** [comment]
Not sure I understand the difference. Currently the branch deletion is immediately for local and then confirm for remote.

My proposal would be to match that behavior where if you select delete tag locally, we would prompt the same dialog to confirm remote deletion. If you have a different flow in mind let me know. I’m just trying to maintain consistency.

**Comment by richardtop** [comment]
Yes, now I get it. I think, what you're proposing is the best way to handle it.

**Comment by sesam** [comment]
#good-first-issue
